test/lsp-io-tests.el

```diff
@@ -58,5 +58,18 @@
 		(should (equal (lsp--parser-read p "\"somedata\":2}")
 									 '("{\"somedata\":2}")))))
 
+(ert-deftest lsp--parser-read--multiple-chunks-ignored ()
+  (let* ((log '())
+         (p (make-lsp--parser :workspace lsp--test-workspace))
+         (lsp--parser-reset p)
+         (filter (lsp--parser-make-filter p '("IGNORE_ME"))))
+    (cl-letf (((symbol-function 'lsp--parser-on-message)
```

Comment:
`cl-letf*` is a generalization of `let*`, so you can use a single `cl-letf*` form here.